daprdocs/content/en/dotnet-sdk-docs/dotnet-client/_index.md

```diff
@@ -199,6 +199,21 @@ await foreach (var items in subscribeConfigurationResponse.Source.WithCancellati
 }
 ```
 
+### Call and run a workflow
```

Comment:
I would go with "Manage workflow instances (Alpha)" or "Run, query, and terminate workflows (Alpha)", though over time we will add new operations here, like suspend/resume/etc.